Shokofin/Shokofin.csproj

```diff
@@ -7,6 +7,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="AsyncKeyedLock" Version="6.4.2" />
```

Comment:
Have you tested that this commit works in Jellyfin? It may build fine, but still not work properly in Jellyfin because of a missing DLL reference, and I don't see any changes to copy the DLL to the output directory.

Reply:
Jellyfin is using this library in the unstable builds and will be in 10.9

Reply:
Seems like it. Can you rebase this onto the 10.9 branch?